tuna/load_job.py

```diff
@@ -110,7 +110,6 @@ def parse_args():
       '--session_id',
       action='store',
       required=True,
-      type=int,
```

Comment:
Why remove int?

Reply:
cause u cant have action=store and define a type with jsonargparse ArgParser.

Reply:
Delete store and keep int like it is in update_golden?